src/types/entities.ts

```diff
@@ -30,3 +30,78 @@ export interface User {
   birthday: string | null;
   bioHtml: string | null;
 }
+
+export enum WorkRatings {
+  GENERAL_AUDIENCES = "General Audiences",
+  TEEN_AND_UP_AUDIENCES = "Teen And Up Audiences",
+  MATURE = "Mature",
+  EXPLICIT = "Explicit",
+}
+
+export enum WorkCategory {
+  FF = "F/F",
+  FM = "F/M",
+  GEN = "Gen",
+  MM = "M/M",
+  MULTI = "Multi",
+  OTHER = "Other",
+}
+
+export enum WorkWarningStatus {
```

Comment:
I think we're missing the option for it to be 'No Archive Warnings Apply'?

Reply:
Added!